### DISCUSSION TITLE
PSS Revive

### DISCUSSION THREAD

**Discussion by gigli69** [BobanSpasic/PSS-Revive#6] [General]
Hello,

Thanks for this software revival!
I try it with my PSS680 under Linux Mate, it is working, except sqlite (sqklit3 is installed. I'd to understand why.

Regards

**Comment by BobanSpasic** [comment]
If the program is starting, it means it found sqlite. What does not work exactly?

**Reply by gigli69** [reply]
Under Linux, when launching the program, the error "Can not load SQLite client library "libsqlite3.so". Check your installation".

Then I click OK, and the program is launching, and can communicate with the PSS, but without loading/saving feature.

**Reply by BobanSpasic** [reply]
Loading/Saving banks as files should be working even without SQLite. SQLite is used to store your collection in a database.
Could you post the output of the following command (run in terminal):
`
apt-show-versions sqlite3
`

In the case that one is not working, please try:
`
sqlite3 --version
`
and post the output here.

**Reply by gigli69** [reply]
3.31.1-4ubuntu0.5

**Reply by BobanSpasic** [reply]
Try this one too, please:
`
aptitude show sqlite3
`
Here is my output:
```
Package: sqlite3
Version: 3.31.1-4ubuntu0.5  
State: installed  
Automatically installed: no  
Multi-Arch: foreign  
Priority: optional  
Section: misc  
Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>  
Architecture: amd64  
Uncompressed Size: 2,803 k  
Depends: libc6 (>= 2.29), libreadline8 (>= 6.0), zlib1g (>= 1:1.2.0), libsqlite3-0 (= 3.31.1-4ubuntu0.5)  
Suggests: sqlite3-doc  
Conflicts: sqlite3:i386  
Provides: sqlite3:i386 (= 3.31.1-4ubuntu0.5)  
Provided by: sqlite3:i386 (3.31.1-4), sqlite3:i386 (3.31.1-4ubuntu0.5)  
Description: Command line interface for SQLite 3  
SQLite is a C library that implements an SQL database engine. Programs that link with the SQLite library can have SQL 
database access without running a separate RDBMS process.  
Homepage: https://www.sqlite.org/  
```
This one will tell me if you have a 64-bit or 32-bit version installed.
